packages/lesswrong/server/postgresView.ts

```diff
@@ -44,25 +60,32 @@ export const createPostgresView = (
     }
   }
   const view = new PostgresView(
+    name,
+    refreshInterval,
     createViewQuery,
     createIndexQueries,
     refreshQuery,
   );
   postgresViews.push(view);
 }
 
-export const ensurePostgresViewsExist = async (db = getSqlClientOrThrow()) => {
+export const ensurePostgresViewsExist = async (
+  db = getSqlClientOrThrow(),
+  addCronJob?: (options: CronJobSpec) => void,
```

Comment:
```suggestion
  /** Dependency injected to avoid cycle. Should always be passed in unless in testing. */
  addCronJob?: (options: CronJobSpec) => void,
```